docs/protocol/extensions.rst

```diff
@@ -11,6 +11,9 @@ Under construction.
    extensions/filters/v1.0
    extensions/complex-dtypes/v1.0
    extensions/datetime-dtypes/v1.0
+   extensions/object-dtypes/v1.0
```

Comment:
Mostly as a side note as we work out the process, we may want to lower these numbers depending in our confidence and the intended interpretation.